src/app.ts

```diff
@@ -28,4 +28,11 @@ app.post('/login', (req, res) => {
   res.redirect('/admin')
 })
 
-app.listen(port, () => console.log(`app listening on port ${port}!`))
+const listener = (): void =>{
+  if(!process.env.ADMIN){
+    return console.log('Admin UI/API is turned off')
+  }
+  app.listen(port, () => console.log(`app listening on port ${port}!`))
```

Comment:
could you also change `package.json` so that `yarn start` will set ENV variables? This way, ppl won't be confused when this gets merged in.